rln-research/2022-07-22-rln-relay-journey.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+name:  ""
+title: ""
+date: 2022-10-18 10:00:00
+author: sanaztaheri
+published: true
+permalink: /waku-rln-relay-journey
+categories: research
+summary: The current state of the research and development of the WAKU-RLN-RELAY project.
```

Comment:
This summary could be better:

- Doesn't help someone who doesn't already know what RLN or RLN RELAY is.

- It also doesn't hint at what the main delta is from last post (e.g. cross client on-chain testnet, etc) - can use a few keywords in a list to give people a better view of what the post about.

The first XX words of the summary shows up in e.g. Twitter view

Reply:
> Doesn't help someone who doesn't already know what RLN or RLN RELAY is.

Fair point.

> It also doesn't hint at what the main delta is from last post (e.g. cross client on-chain testnet, etc) - can use a few keywords in a list to give people a better view of what the post about.

Sure.

> The first XX words of the summary shows up in e.g. Twitter view

Noted.

Please check the following commit https://github.com/vacp2p/research/pull/143/commits/5e3ab814150340248272524c2473c4ca60346856